### DISCUSSION TITLE
Dropdown Values Duplicating in Modal When Reopened (PrimeFaces 13 vs 8)

### DISCUSSION THREAD

**Discussion by arcavus** [primefaces/community#3170] [PrimeFaces] [answered]
Environment:
- PrimeFaces Version: 13.x
- Previously Working Version: 8.x
- Component: p:selectOneMenu within modal

Issue Description:
When reopening a modal dialog, dropdown (selectOneMenu) values are duplicating. The component is correctly configured, and the issue only occurs in version 13. In version 8, it worked perfectly.

Specific Observations:
- First modal opening: Dropdown works correctly
- Subsequent modal openings: Dropdown values multiply
- Component configuration remains unchanged
- Reproduction consistent across attempts

Possible Causes I've Considered:
1. Potential changes in PrimeFaces component rendering logic
2. Modifications in selection and item management
3. Potential conflict with view state handling

Request:
Can anyone provide insights into this behavioral change or suggest a workaround to prevent value duplication?  
Second problem min max values does not work properly input number element.
![Untitled2](https://github.com/user-attachments/assets/340ed885-2c2a-441e-ae97-0499fa9f0deb)
![Untitled1](https://github.com/user-attachments/assets/ce5efd78-cf7e-4125-a440-d62d7ee0d0a6)

**Comment by tandraschko** [comment]
did you try the newest 14.x versions? 
if it doesnt work there, we propably need a reproducer.

**Reply by arcavus** [reply]
My java version 1.8 and not compability  14.x version.

**Comment by melloware** [comment]
Please provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.

**Comment by melloware** [comment]
Closing until OP provides a reproducer

**Comment by arcavus** [comment, marked as answer]
I found answer my problem reason of duplicating select item values that append to body attribute. İf remove this attribute work correctly dropdown. @appendTo and dynamic attribute maybe reason this anomally.
